From 7c468d8ea83d5a106904c98af5414236aa40d847 Mon Sep 17 00:00:00 2001
From: Markus Niebel <Markus.Niebel@tq-group.com>
Date: Thu, 18 Apr 2019 13:37:05 +0200
Subject: [PATCH] arm64: dt: rename tqma8qxps -> tqma8xqps

This rename should (hopefully) correct the naming mess:

module is TQMa8XxS (X for the i.MX8 X family, x for CPU variant S means SMARC)

We rename / move here:

- file names
- compatible names
- model names

Signed-off-by: Markus Niebel <Markus.Niebel@tq-group.com>
---
 arch/arm64/boot/dts/freescale/Makefile        |  6 +++---
 ...dts => fsl-imx8dx-tqma8xds-mb-smarc-2.dts} |  8 +++----
 .../dts/freescale/fsl-imx8dx-tqma8xds.dtsi    | 21 +++++++++++++++++++
 ...s => fsl-imx8dxp-tqma8xdps-mb-smarc-2.dts} |  8 +++----
 ...a8dxps.dtsi => fsl-imx8dxp-tqma8xdps.dtsi} |  6 +++---
 ...s => fsl-imx8qxp-tqma8xqps-mb-smarc-2.dts} |  6 +++---
 ...a8qxps.dtsi => fsl-imx8qxp-tqma8xqps.dtsi} |  4 ++--
 7 files changed, 40 insertions(+), 19 deletions(-)
 rename arch/arm64/boot/dts/freescale/{fsl-imx8dx-tqma8dxs-mb-smarc-2.dts => fsl-imx8dx-tqma8xds-mb-smarc-2.dts} (75%)
 create mode 100644 arch/arm64/boot/dts/freescale/fsl-imx8dx-tqma8xds.dtsi
 rename arch/arm64/boot/dts/freescale/{fsl-imx8dxp-tqma8dxps-mb-smarc-2.dts => fsl-imx8dxp-tqma8xdps-mb-smarc-2.dts} (75%)
 rename arch/arm64/boot/dts/freescale/{fsl-imx8dxp-tqma8dxps.dtsi => fsl-imx8dxp-tqma8xdps.dtsi} (81%)
 rename arch/arm64/boot/dts/freescale/{fsl-imx8qxp-tqma8qxps-mb-smarc-2.dts => fsl-imx8qxp-tqma8xqps-mb-smarc-2.dts} (80%)
 rename arch/arm64/boot/dts/freescale/{fsl-imx8qxp-tqma8qxps.dtsi => fsl-imx8qxp-tqma8xqps.dtsi} (87%)

diff --git a/arch/arm64/boot/dts/freescale/Makefile b/arch/arm64/boot/dts/freescale/Makefile
index 5b6dd5277332..67dd38af046f 100644
--- a/arch/arm64/boot/dts/freescale/Makefile
+++ b/arch/arm64/boot/dts/freescale/Makefile
@@ -92,17 +92,17 @@ dtb-$(CONFIG_ARCH_FSL_IMX8QXP) += fsl-imx8qxp-lpddr4-arm2.dtb \
 				  fsl-imx8dx-tqma8xd-mba8xx.dtb \
 				  fsl-imx8dx-tqma8xd-mba8xx-lvds-tm070jvhg33.dtb \
 				  fsl-imx8dx-tqma8xd-mba8xx-pcie-gen1.dtb \
-				  fsl-imx8dx-tqma8dxs-mb-smarc-2.dtb \
+				  fsl-imx8dx-tqma8xds-mb-smarc-2.dtb \
 				  fsl-imx8dxp-lpddr4-arm2.dtb \
 				  fsl-imx8dxp-tqma8xdp-mba8xx.dtb \
 				  fsl-imx8dxp-tqma8xdp-mba8xx-lvds-tm070jvhg33.dtb \
 				  fsl-imx8dxp-tqma8xdp-mba8xx-pcie-gen1.dtb \
-				  fsl-imx8dxp-tqma8dxps-mb-smarc-2.dtb \
+				  fsl-imx8dxp-tqma8xdps-mb-smarc-2.dtb \
 				  fsl-imx8qxp-tqma8xqp-mba8xx.dtb \
 				  fsl-imx8qxp-tqma8xqp-mba8xx-lvds-tm070jvhg33.dtb \
 				  fsl-imx8qxp-tqma8xqp-mba8xx-pcie-gen1.dtb \
 				  fsl-imx8qxp-tqma8xqp-mba8xx-dsilvds.dts \
-				  fsl-imx8qxp-tqma8qxps-mb-smarc-2.dtb
+				  fsl-imx8qxp-tqma8xqps-mb-smarc-2.dtb
 dtb-$(CONFIG_ARCH_FSL_IMX8MQ) += fsl-imx8mq-ddr3l-arm2.dtb \
 				 fsl-imx8mq-ddr4-arm2.dtb \
 				 fsl-imx8mq-ddr4-arm2-gpmi-nand.dtb \
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8dx-tqma8dxs-mb-smarc-2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8dx-tqma8xds-mb-smarc-2.dts
similarity index 75%
rename from arch/arm64/boot/dts/freescale/fsl-imx8dx-tqma8dxs-mb-smarc-2.dts
rename to arch/arm64/boot/dts/freescale/fsl-imx8dx-tqma8xds-mb-smarc-2.dts
index d1a8b2edf5df..740dea565f64 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8dx-tqma8dxs-mb-smarc-2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8dx-tqma8xds-mb-smarc-2.dts
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 -2019 TQ Systems GmbH
+ * Copyright 2019 TQ Systems GmbH
  *
  * This program is free software; you can redistribute it and/or
  * modify it under the terms of the GNU General Public License
@@ -15,10 +15,10 @@
 /dts-v1/;
 
 #include <dt-bindings/net/ti-dp83867.h>
-#include "fsl-imx8dx-tqma8dxs.dtsi"
+#include "fsl-imx8dx-tqma8xds.dtsi"
 #include "mb-smarc-2-common.dtsi"
 
 / {
-	model = "TQ Systems i.MX8DX TQMa8DXS on MB-SMARC-2";
-	compatible = "tq,tqma8dxs-mb-smarc-2", "tq,tqma8dxs", "fsl,imx8dx";
+	model = "TQ Systems i.MX8DX TQMa8XDS on MB-SMARC-2";
+	compatible = "tq,tqma8xds-mb-smarc-2", "tq,tqma8xds", "fsl,imx8dx";
 };
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8dx-tqma8xds.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8dx-tqma8xds.dtsi
new file mode 100644
index 000000000000..01d01a51b588
--- /dev/null
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8dx-tqma8xds.dtsi
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2019 TQ Systems GmbH
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+
+#include "fsl-imx8dx.dtsi"
+#include "tqma8xxs-common.dtsi"
+
+/ {
+	model = "TQ Systems i.MX8DX TQMa8XDS";
+	compatible = "tq,tqma8xds", "fsl,imx8qxp";
+};
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8dxp-tqma8dxps-mb-smarc-2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8dxp-tqma8xdps-mb-smarc-2.dts
similarity index 75%
rename from arch/arm64/boot/dts/freescale/fsl-imx8dxp-tqma8dxps-mb-smarc-2.dts
rename to arch/arm64/boot/dts/freescale/fsl-imx8dxp-tqma8xdps-mb-smarc-2.dts
index 1bcc70c5f267..55c4d559e30f 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8dxp-tqma8dxps-mb-smarc-2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8dxp-tqma8xdps-mb-smarc-2.dts
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 -2019 TQ Systems GmbH
+ * Copyright 2019 TQ Systems GmbH
  *
  * This program is free software; you can redistribute it and/or
  * modify it under the terms of the GNU General Public License
@@ -15,10 +15,10 @@
 /dts-v1/;
 
 #include <dt-bindings/net/ti-dp83867.h>
-#include "fsl-imx8dxp-tqma8dxps.dtsi"
+#include "fsl-imx8dx-tqma8xds.dtsi"
 #include "mb-smarc-2-common.dtsi"
 
 / {
-	model = "TQ Systems i.MX8DXP TQMa8DXPS on MB-SMARC-2";
-	compatible = "tq,tqma8dxps-mb-smarc-2", "tq,tqma8dxps", "fsl,imx8dxp";
+	model = "TQ Systems i.MX8DXP TQMa8XDPS on MB-SMARC-2";
+	compatible = "tq,tqma8xdps-mb-smarc-2", "tq,tqma8xdps", "fsl,imx8dxp";
 };
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8dxp-tqma8dxps.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8dxp-tqma8xdps.dtsi
similarity index 81%
rename from arch/arm64/boot/dts/freescale/fsl-imx8dxp-tqma8dxps.dtsi
rename to arch/arm64/boot/dts/freescale/fsl-imx8dxp-tqma8xdps.dtsi
index 064767542afe..23e966fa3121 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8dxp-tqma8dxps.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8dxp-tqma8xdps.dtsi
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 - 2019 TQ Systems GmbH
+ * Copyright 2019 TQ Systems GmbH
  *
  * This program is free software; you can redistribute it and/or
  * modify it under the terms of the GNU General Public License
@@ -16,6 +16,6 @@
 #include "tqma8xxs-common.dtsi"
 
 / {
-	model = "TQ Systems i.MX8DXP TQMa8DXPS";
-	compatible = "tq,tqma8dxps", "fsl,imx8dxp";
+	model = "TQ Systems i.MX8DXP TQMa8XDPS";
+	compatible = "tq,tqma8xdps", "fsl,imx8qxp";
 };
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-tqma8qxps-mb-smarc-2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-tqma8xqps-mb-smarc-2.dts
similarity index 80%
rename from arch/arm64/boot/dts/freescale/fsl-imx8qxp-tqma8qxps-mb-smarc-2.dts
rename to arch/arm64/boot/dts/freescale/fsl-imx8qxp-tqma8xqps-mb-smarc-2.dts
index e3e2eb290bd5..f5e2deb0d89c 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-tqma8qxps-mb-smarc-2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-tqma8xqps-mb-smarc-2.dts
@@ -15,10 +15,10 @@
 /dts-v1/;
 
 #include <dt-bindings/net/ti-dp83867.h>
-#include "fsl-imx8qxp-tqma8qxps.dtsi"
+#include "fsl-imx8qxp-tqma8xqps.dtsi"
 #include "mb-smarc-2-common.dtsi"
 
 / {
-	model = "TQ Systems i.MX8QXP TQMa8QXPS on MB-SMARC-2";
-	compatible = "tq,tqma8qxps-mb-smarc-2", "tq,tqma8qxps", "fsl,imx8qxp";
+	model = "TQ Systems i.MX8QXP TQMa8XQPS on MB-SMARC-2";
+	compatible = "tq,tqma8xqps-mb-smarc-2", "tq,tqma8xqps", "fsl,imx8qxp";
 };
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-tqma8qxps.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-tqma8xqps.dtsi
similarity index 87%
rename from arch/arm64/boot/dts/freescale/fsl-imx8qxp-tqma8qxps.dtsi
rename to arch/arm64/boot/dts/freescale/fsl-imx8qxp-tqma8xqps.dtsi
index d62318ba029b..c0734ad73643 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-tqma8qxps.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-tqma8xqps.dtsi
@@ -16,6 +16,6 @@
 #include "tqma8xxs-common.dtsi"
 
 / {
-	model = "TQ Systems i.MX8QXP TQMa8QXPS";
-	compatible = "tq,tqma8qxps", "fsl,imx8qxp";
+	model = "TQ Systems i.MX8QXP TQMa8XQPS";
+	compatible = "tq,tqma8xqps", "fsl,imx8qxp";
 };
